src/redux-store/AuthOperations/AuthOperations.js

```diff
@@ -68,13 +54,19 @@ export const logIn = createAsyncThunk('auth/login', async userData => {
 
 export const logOut = createAsyncThunk('auth/logout', async () => {
   try {
-    await axios.post('/api/authentication/logout');
+    await sendRequest({
+      method: 'post',
+      endpoint: 'logout',
+    });
+
     token.unset();
-  } catch (e) {
-    console.log(e.message);
+  } catch (error) {
+    throw new Error(error.message);
   }
 });
 
+// TODO fetchCurrentUser, updateUser and other requests shuold be update when we'll take the Profile component in dev
```

Comment:
Whu didn't you apply changes for all requests?

Reply:
Because these components don't work still and I don't know how these requests will be working in the future

Reply:
Done, but sendDataCountryToBackend request is still under TODO comment